changefeedtrigger/changefeedSample.py

```diff
@@ -0,0 +1,81 @@
+from .settings import *
+from azure.storage.blob.changefeed import ChangeFeedClient
+from azure.storage.blob import BlobServiceClient
+import azure.functions as func
+import logging
+from azure.core.exceptions import ResourceNotFoundError
+
+
+def main(mytimer: func.TimerRequest) -> None:
+
+    # create blob service client to access connection string
+    blob_service_client = BlobServiceClient.from_connection_string(CONNECTION_STRING)
+    change_feed_client = ChangeFeedClient.from_connection_string(CONNECTION_STRING)
+
+    # create or get container to store cursor
+    container_name = "cursorstoragecontainer"
+    try:
+        token_container_client = blob_service_client.create_container(container_name)
+    except:
+        token_container_client = blob_service_client.get_container_client(container_name)
+
+    # call to method
+    events = get_events(token_container_client, change_feed_client)
+    events = filter_events(events)
+ 
+    # logging filtered event output
+    if len(events) == 0:
+        logging.info("\nNo events matching filters on current page...")
+    else:
+        logging.info("\nRecents event(s) matching filters on current page:")
+        count = 1
+        for e in events:
+            logging.info("%d: %s\n" % (count, e))
+            count += 1
+
+
+def get_events(token_client, cf_client):
+    # this method uses the continuation token to iterate on events and updates the token
+
+    # check if continuation token already exists
+    blob_name = "cursorBlob"
+    blob_client = token_client.get_blob_client(blob_name)
+    try:
+        blob_client.get_blob_properties
```

Comment:
>get_blob_properties [](start = 20, length = 19)

this should be the bug.... you missed ()

Reply:
I think probably blob_client.get_blob_properties() is not necessary, download blob should also throw the same error if the blob isn't found. Can you try that

---
In reply to: [470901607](https://github.com/wastore/azure-storage-samples-for-python/pull/7#discussion_r470901607) [](ancestors = 470901607)

Reply:
You are correct, this line is not needed! I will delete it